src/coreComponents/physicsSolvers/SolverBase.hpp

```diff
@@ -734,6 +734,7 @@ class SolverBase : public ExecutableGroup
   virtual bool registerCallback( void * func, const std::type_info & funcType ) final override;
 
   SolverStatistics & getSolverStatistics() { return m_solverStatistics; }
+  SolverStatistics const & getSolverStatistics() const { return m_solverStatistics; }
```

Comment:
Who's using the non-const version?
If you can, can you remove it or use a setter instead?

Reply:
It is used at `src/coreComponents/physicsSolvers/multiphysics/CoupledSolver.hpp:431, 491, 506`, I think both version are useful.